### PR TITLE
Updated after moving to GH.

### PR DESCRIPTION
**Pull request by bwplotka**
Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Review by onprem — APPROVED** [review]
LGTM!

File: CODEOWNERS [hunk 1]
```diff
@@ -1,6 +1,4 @@
-- bplotka
-- fbranczy
+- bwplotka
 - kakkoyun
-- lserven
-- mloibl
-- kgeorgie
\ No newline at end of file
+- squat
+- onprem
```
Comment (squat) [inline CODEOWNERS:4]: maybe we can add @spaparaju  too

File: README.md [hunk 2]
```diff
@@ -1,24 +1,25 @@
-# observatorium/configuration
+# Red Hat Observability Service
 
-This projects holds all the configuration files for our internal Observatorium deployments.
+This project holds the configuration files for our internal Red Hat Observability Service based using [Observatorium](https://github.com/observatorium/observatorium).
```
Comment (squat) [inline README.md:3]:
```suggestion
This project holds the configuration files for our internal Red Hat Observability Service based on [Observatorium](https://github.com/observatorium/observatorium).
```

**Review by squat — COMMENTED** [review]
small nits, lgtm overall!

**Review by kakkoyun — APPROVED** [review]
🚀

**Pull request merged**